charts/velero/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.5.3
 description: A Helm chart for velero
 name: velero
-version: 2.15.0
+version: 2.15.1
```

```suggestion
version: 2.16.0
```
because it is a feature addition and not a fix
besides that LGTM

---

Since I was trying to fix an issue so I think it's a fix.

---

Never mind, agree with you.